storage/src/robocorp/storage/__init__.py

```diff
@@ -42,13 +42,36 @@ def _get_client() -> "AssetsClient":
     return AssetsClient(workspace, endpoint, token)
 
 
-def list_assets() -> List[str]:
+def list_assets(page_limit: Optional[int] = 100) -> List[str]:
     """List all the existing assets.
 
+    Args:
+        page_limit: How many assets to retrieve per request
```

Comment:
```suggestion
        page_limit: How many assets to retrieve per requested page (increase it if your asset isn't returned)
```
Just in case we want better rewording.